docker/Dockerfile

```diff
@@ -6,35 +6,35 @@ RUN apt-get -qq update --yes \
     ca-certificates wget locales \
  && rm -rf /var/lib/apt/lists/*
 
-# Set locale
-RUN echo "pt_BR.UTF-8 UTF-8" > /etc/locale.gen
-RUN locale-gen pt_BR.UTF-8
-RUN update-locale pt_BR.UTF-8
-ENV LC_ALL pt_BR.UTF-8
-ENV LC_ALL=C.UTF-8
-ENV LANG=C.UTF-8
+#Set locale
+RUN sed -i -e "s/# pt_BR.*/pt_BR.UTF-8 UTF-8/" /etc/locale.gen \
+    && dpkg-reconfigure --frontend=noninteractive locales \
+    && update-locale LANG=pt_BR.UTF-8
 
 # Create deploy user
 RUN useradd --shell=/bin/bash --home=/srv/deploy/ --create-home deploy
 
+# Configure conda-channels and install mamba
+RUN conda config --add channels conda-forge \
+  && conda update --all --yes --quiet \
+  && conda install --yes conda-build mamba \
+  && conda clean --all --yes
+
+# Copy environment file to tmp/
 ARG PYTHON_VERSION
 COPY environment-${PYTHON_VERSION}.yml /tmp/environment.yml
 
-RUN /opt/conda/bin/conda config --add channels conda-forge \
-  && /opt/conda/bin/conda update --all --yes --quiet \
-  && /opt/conda/bin/conda env update --name base --file /tmp/environment.yml \
-  && /opt/conda/bin/conda install --yes conda-build \
-  && /opt/conda/bin/conda clean --all --yes
-
-ENV PATH="/opt/conda/bin:${PATH}"
+# Use environment to update the env base
+RUN mamba env update --file /tmp/environment.yml --name base \
+  && mamba clean --all --yes
 
-# copy code to the container
-# Change the permissions for the user home directory
+# Copy AlertaDengue into home directory and fix permissions
 RUN mkdir /srv/deploy/logs
 COPY --chown=deploy AlertaDengue  /srv/deploy/AlertaDengue
 
 WORKDIR /srv/deploy/AlertaDengue
 
 EXPOSE 8000
 USER deploy
-CMD ["/srv/deploy/AlertaDengue/runwsgi.sh"]
+CMD [“conda”, “env”, “create”, “source activate base"]
```

Comment:
does it really need this?

Reply:
It is not necessary but only ensures that the container goes up with the conda base activated...

Reply:
In our case, it doesn't make sense even to have the entrypoint or even the conda env activated to run the image, i fix this in another commit. 
Thanks